save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -294,14 +298,13 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
             }
         },
         getAdditionalDependencies = {
-            arrayOf(it.status, it.testSuite)
+            arrayOf(it.filters.status, it.filters.fileName, it.filters.testSuite, it.filters.tag)
         }
     )
 
     init {
         state.executionDto = null
-        state.status = null
-        state.testSuite = null
+        state.filters = TestExecutionFilters(null, null, null, null)
```

Comment:
You have `TestExecutionFilters.empty` for it, don't you?

Reply:
Done

Reply:
What do you mean by saying that?

Reply:
Changed empty values fileName, testSuite, tags in TestExecutionFilters